frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -67,6 +77,33 @@ export const MonitoringAlerts: React.FC<props> = ({ match, rules, filters, listS
     dispatch(monitoringLoaded('devAlerts', thanosAlertsAndRules.alerts, 'dev'));
   }, [dispatch, thanosAlertsAndRules]);
 
+  const alertStateFilter: RowFilter = {
+    filter: ruleAlertStateFilter,
+    filterGroupName: t('public~Alert State'),
+    isMatch: ruleHasAlertState,
+    items: [
+      { id: AlertStates.Firing, title: t('public~Firing') },
+      { id: AlertStates.Pending, title: t('public~Pending') },
+      { id: AlertStates.Silenced, title: t('public~Silenced') },
+      { id: AlertStates.NotFiring, title: t('public~Not Firing') },
+    ],
+    type: 'alerting-rule-has-alert-state',
+  };
+
+  const severityRowFilter: RowFilter = {
+    filterGroupName: t('public~Severity'),
+    items: [
+      { id: AlertSeverity.Critical, title: t('public~Critical') },
+      { id: AlertSeverity.Warning, title: t('public~Warning') },
+      { id: AlertSeverity.Info, title: t('public~Info') },
+      { id: AlertSeverity.None, title: t('public~None') },
+    ],
+    reducer: ({ labels }: Alert | Rule) => labels?.severity,
+    type: 'alert-severity',
+  };
+
+  const alertFilters: RowFilter[] = [alertStateFilter, severityRowFilter];
+
```

Comment:
It looks like these filters are duplicates of those defined in `alerting.tsx`. I'd prefer to just define them once and leave this importing them. I guess that will mean using `i18next.t` instead of `useTranslation` when defining them.

Reply:
i18next.t is iffy in row filters. Using it causes missingKey errors that will break the CI tests. Sam's against passing the TFunction around as an argument, so this was the only other option.

Reply:
I wasn't aware that `i18next.t` is iffy 😢 

Although we are mostly avoiding passing around `t`, it seems like it would be better than duplicating blocks of code? WDYT @spadgett?

Reply:
Figured it out! I'll open a tech debt story to clean up all the filters passing the TFunction I noticed in the codebase. Thanks for your patience.